lib/utils/fs/writeFileSync.js

```diff
@@ -1,16 +1,17 @@
 'use strict';
 
+const fse = require('./fse');
 const path = require('path');
+const jc = require('json-cycle');
 const YAML = require('js-yaml');
-const fse = require('./fse');
 
 function writeFileSync(filePath, conts) {
   let contents = conts || '';
 
   fse.mkdirsSync(path.dirname(filePath));
 
   if (filePath.indexOf('.json') !== -1 && typeof contents !== 'string') {
-    contents = JSON.stringify(contents, null, 2);
+    contents = jc.stringify(contents);
```

Comment:
FWIW, the 2nd and 3rd parameters are not recognized by `json-cycle` so it doesn't yet make sense to pass them